src/common/LRUCache.cpp

```diff
@@ -61,6 +67,20 @@ class LRUCache {
         return deleteVector;
     }
 
+    std::vector<key_t> removeNumOfKeys(size_t num) {
```

Comment:
Only key is returned? The value can be lost.
Currently, my key-value pair is blockId-BlockInfo